hotfixes/4.3.15.cfg

```diff
@@ -0,0 +1,12 @@
+[buildout]
+
+hotfix-eggs =
+
```

Comment:
@jone I just run the update. It removed some hotfixes. Can you check the result?